server/schema/type.js

```diff
@@ -0,0 +1,86 @@
+const pageInfoType = `
+  type PageInfo {
+    currentPage: Int!
+    totalPages: Int!
+    hasNextPage: Boolean!
+    hasPreviousPage: Boolean!
+    totalItems: Int!
+  }
+`;
+
+const bookType = `
+    type Book {
```

Comment:
Any difference if we use node instead of type?

Reply:
Not really, but I was not aware about Node.